### DISCUSSION TITLE
Any DIfference in Google Servers?

### DISCUSSION THREAD

**Discussion by Scott-S87** [derekhe/msfs2020-google-map#101] [Q&A]
Is there any known differences between the available Google servers (mt1, mt2, or mt3)? Is there any noticeable speed differences? Differences in age of the imagery (perhaps one server has newer images than the others?).  I do notice that on Google Maps, there are certain new construction features that appear, but do not when I fly over those areas in MSFS when using the Google Maps application. Normally I use mt2, and it works just fine for me! 

![image](https://user-images.githubusercontent.com/58869287/142699946-1a383094-d687-4bba-8bfd-1cd5b9165ebd.png)

Thanks! Awesome, amazing product by the way!! 
Scott

**Comment by derekhe** [comment]
They are same but for load balance.

**Reply by Scott-S87** [reply]
Okay cool, thank you!!
